sites/default/menu_data.json

```diff
@@ -988,41 +994,41 @@
         "global_req" : "addr_label_type"
     } ],
     "requirement" : 0
-}, {
+}, { 
     "label" : "QA Measures",
     "menu_id" : "mips",
-    "children" : [ {
+    "children" : [ { 
```

Comment:
How did the extra white pace come in?  Was that from your editor, or was it generated by the menu editor?  If it was from the menu editor, we should probably leave that as a standard.